artifact-registry/src/sign.rs

```diff
@@ -0,0 +1,21 @@
+use std::fmt::{Debug, Formatter};
```

Comment:
This file appears to be unused. If you do not plan to use it, please delete it.

Reply:
Actually it's used [here](https://github.com/yoshidan/google-cloud-rust/pull/207/files#diff-6ae5df621ad47f7a1aaae4f5cf0069bca83edf946e3eb47b15a6c78cce88a3d5R44) and [here](https://github.com/yoshidan/google-cloud-rust/pull/207/files#diff-6ae5df621ad47f7a1aaae4f5cf0069bca83edf946e3eb47b15a6c78cce88a3d5R51)

Reply:
Do you plan to use `default_google_access_id` ,`default_sign_by` and `project_id` for anything in `google-artifact-registry`? If not, please remove them.

`google-cloud-storage` required these values for signed URL generation.
However, `google-cloud-pubsub` and `google-cloud-spanner` and `google-cloud-bigquery` do not define these values because they are unnecessary.
https://github.com/yoshidan/google-cloud-rust/blob/62c4deebff980098807b95aa773403eaa13a0aa1/spanner/src/client.rs#L111
https://github.com/yoshidan/google-cloud-rust/blob/62c4deebff980098807b95aa773403eaa13a0aa1/pubsub/src/client.rs#L60
https://github.com/yoshidan/google-cloud-rust/blob/62c4deebff980098807b95aa773403eaa13a0aa1/bigquery/src/client.rs#L112

Reply:
You are right, my bad, let's try to remove it all and see how it goes later on :)